src/components/button/button.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import Button, { ButtonType } from '.'
```

Comment:
O padrão pro nome de enum é full maiusculo, por que você está importando como minusculo? 
```suggestion
import Button, { BUTTON_TYPE } from '.'
```

Reply:
commit: 4b4354dcf409be5c9ef4043d1e820fee87356934